openedx/features/course_experience/utils.py

```diff
@@ -51,7 +51,7 @@ def recurse_mark_scored(block):
         is_scored = block.get('has_score', False) and block.get('weight', 1) > 0
         # Use a list comprehension to force the recursion over all children, rather than just stopping
         # at the first child that is scored.
-        children_scored = any(recurse_mark_scored(child) for child in block.get('children', []))
+        children_scored = any(tuple(recurse_mark_scored(child) for child in block.get('children', [])))
```

Comment:
does it work with list()?

Reply:
can you change it to list? or does it make more sense as tuple?

Reply:
Creating tuples is faster than list